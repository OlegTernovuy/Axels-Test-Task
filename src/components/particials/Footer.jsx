import React from "react";
import { StyledFooter } from "../../styled/particials/StyledFooter";

const Footer = () => {
  return (
    <StyledFooter>
      <span>Oleh Ternovyi 2024</span>
    </StyledFooter>
  );
};

export default Footer;
