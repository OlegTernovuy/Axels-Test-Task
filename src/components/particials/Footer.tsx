import { StyledFooter } from '../../styled/particials/StyledFooter';

const Footer = () => (
    <StyledFooter>
        <span>Oleh Ternovyi 2024</span>
    </StyledFooter>
);

export default Footer;
