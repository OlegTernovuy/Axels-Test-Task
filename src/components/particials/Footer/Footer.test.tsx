import renderer from 'react-test-renderer';
import Footer from './Footer';

describe('Footer snapshot component', () => {
    it('Matches Snapshot', () => {
        const footerComponent = renderer.create(<Footer />).toJSON();
        expect(footerComponent).toMatchSnapshot();
    });
});
