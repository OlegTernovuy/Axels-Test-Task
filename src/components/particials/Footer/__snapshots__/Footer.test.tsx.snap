// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer snapshot component Matches Snapshot 1`] = `
<footer
  className="sc-aYaIB fUUDmR"
>
  <span>
    Oleh Ternovyi 2024
  </span>
</footer>
`;
